src/devtools/main.ts

```diff
@@ -362,12 +367,35 @@ async function initialize(): Promise<void>{
     }
 }
 
-function handleMcdevSBCommand(){
+function handleMCDevSBCommand(){
     editorOutput.showOuputChannel();
 }
 
+function getMCDevRelativePathComponents(relativePath: string): DevToolsPathComponents {
+    const [
+        credentialName,
+        businessUnit,
+        metadataType,
+        ...keys
+    ]: string[] = relativePath.split("/");
+    return { credentialName, businessUnit, metadataType, keys };
+}
+
+function logUnsupportedMtdtTypeNotification(action: string, unsupportedMtdtTypes: string | string[]){
+    [unsupportedMtdtTypes]
+    .flat()
+    .forEach((metadataType: string) => {
+        log(
+            "info", 
```

Comment:
```suggestion
            "error", 
```
should be an error, not info.